docker/spec/dependabot/docker/file_fetcher_spec.rb

```diff
@@ -273,8 +273,10 @@
 
       matching_filenames = [
         "other-values.yaml",
+        "other_values.yaml",
         "values.yaml",
         "values2.yaml",
+        "values-other.yaml",
```

Comment:
@brendandburns since you originally wrote these test cases here: https://github.com/dependabot/dependabot-core/commit/7046a3c6668912d64eaf9d1f8c44f5a0ac05f4dd#diff-f117121fabe9e441088126488f34fa0b148aced3b0b99393850768cc6d9534eeR321

Was there a reason you didn't test these two variants as well?

I'm chasing a regex-related bug so wanted to doublecheck whether this was intentional omission or not...

Reply:
After digging some more, pretty sure this wasn't an intentional omission, so safe to add these.